runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -292,25 +296,29 @@ public Token getLogTail() {
      *
      * @param address log address
      */
-    public void prefixTrim(final long address) {
-        log.debug("PrefixTrim[{}]", address);
-        try {
-            layoutHelper(e -> {
-                        e.getLayout().getPrefixSegments(address).stream()
-                                .flatMap(seg -> seg.getStripes().stream())
-                                .flatMap(stripe -> stripe.getLogServers().stream())
-                                .map(e::getLogUnitClient)
-                                .map(client -> client.prefixTrim(address))
-                                .forEach(CFUtils::getUninterruptibly);
-                        return null;    // No return value
-                    }
-            );
-
-            runtime.getSequencerView().trimCache(address);
-
-        } catch (Exception e) {
-            log.error("prefixTrim: Error while calling prefix trimming {}", address, e);
-            throw new UnrecoverableCorfuError("Unexpected error while prefix trimming", e);
+    public void prefixTrim(final Token address) {
+        //TODO(Maithem): adjust sequence by -1 with same epoch
```

Comment:
Do we really need this TODO? For me the current semantics of the address param makes sense.